doc/tutorial/shape_info.txt

```diff
@@ -65,7 +65,7 @@ array([8, 4])
 
 >>> f = theano.function([x,y], z)# Do not take the shape.
 >>> theano.printing.debugprint(f) # doctest: +NORMALIZE_WHITESPACE
-Join [id A] ''   0
+Join{view=-1} [id A] ''   0
```

Comment:
I think I would add a \__str__ method to don't show that. Something like:
```
def __str__(self):
    if self.view == -1:
        return "Join"
    else:
        return super(Join, self).__str__()
```

This won't polute the printing for an option that is rarely used.